battery_limits.py

```diff
@@ -0,0 +1,49 @@
+lang = 'EN'
+
+warning_range_percent = 5
+
+bms_attribute_limits = {'TEMPERATURE': {'min': 0, 'max': 45},
+                        'SOC': {'min': 20, 'max': 80},
+                        'CHARGE_RATE': {'min': 0,'max': 0.8}} 
+
+en_list = ("low_replace_breach","low_replace_warning","normal","high_replace_warning","high_replace_breach")
```

Comment:
If en_list and de_list are shown to the user, they better not have underscores.
Is there a need for the underscores?